### PR TITLE
branding: Replace all Compute@Edge/C@E references with Compute.

### PR DESCRIPTION
**Pull request by kpfleming**

### CODE REVIEW

**Comment by kpfleming** [comment]
I intentionally left the contents of CHANGELOG.md alone since that is historical information, and I also did not modify the binary files in the `testdata` directories as I wasn't sure what would be needed to ensure that tests would still pass.

**Comment by Integralist** [comment]
The CI is failing because gosec released a new version today that no longer respects the `#nosec` directive. It's a simple fix so I can sort that out on Monday.

File: pkg/commands/logtail/root.go [hunk 1]
```diff
@@ -480,7 +480,7 @@ type (
 		stream string
 	}
 
-	// Log defines the message envelope that compute@edge (C@E) wraps the
+	// Log defines the message envelope that Compute (Compute) wraps the
```
Comment (Integralist) [inline pkg/commands/logtail/root.go:483]:
```suggestion
	// Log defines the message envelope that the Compute platform wraps the
```

**Review by Integralist — APPROVED** [review]
LGTM. Just one suggested edit.

**Pull request merged**